stdlib/2and3/datetime.pyi

```diff
@@ -1,31 +1,37 @@
 import sys
 from time import struct_time
-from typing import Optional, SupportsAbs, Tuple, overload
+from typing import AnyStr, Optional, SupportsAbs, Tuple, Union, overload, \
+    ClassVar
 
-MINYEAR = 0
-MAXYEAR = 0
+if sys.version_info >= (3,):
+    _Text = str
+else:
+    _Text = Union[str, unicode]
+
+MINYEAR: int
+MAXYEAR: int
 
 class tzinfo:
     def tzname(self, dt: Optional[datetime]) -> str: ...
     def utcoffset(self, dt: Optional[datetime]) -> Optional[timedelta]: ...
     def dst(self, dt: Optional[datetime]) -> Optional[timedelta]: ...
     def fromutc(self, dt: datetime) -> datetime: ...
 
-class timezone(tzinfo):
-    utc = ...  # type: timezone
-    min = ...  # type: timezone
-    max = ...  # type: timezone
+if sys.version_info >= (3, 2):
+    class timezone(tzinfo):
+        utc: ClassVar['timezone']
+        min: ClassVar['timezone']
+        max: ClassVar['timezone']
 
-    def __init__(self, offset: timedelta, name: str = ...) -> None: ...
-    def __hash__(self) -> int: ...
+        def __init__(self, offset: timedelta, name: str = ...) -> None: ...
+        def __hash__(self) -> int: ...
 
 _tzinfo = tzinfo
-_timezone = timezone
```

Comment:
It looks like these two names are not defined at runtime (at least in Python 3.4).

Reply:
Which names are not defined? timezone is Python 3.2+, but the _timezone alias was not used in the module, which is why removed it.

Reply:
Sorry, I meant that neither ``_tzinfo`` nor ``_timezone`` are defined at runtime. You removed ``_timezone``, so I am wondering maybe we should also remove ``_tzinfo`` and use ``tzinfo`` instead? Or am I missing something?

Reply:
@ilevkivskyi I think the reason `_tzinfo` exists is that `class time` is partially defined as:

```python
class time:
    ...
    @property
    def tzinfo(self) -> Optional[_tzinfo]: ...
    ...
    def replace(self, ..., tzinfo: Optional[_tzinfo] = ...) -> time: ...
```

So using plain `tzinfo` would refer back to the property instead of the type, I think.

Reply:
OK, makes sense. The PR looks good to me, but it would be good if someone else would take a quick look at it.

Reply:
It's fine in general in stubs to add private names starting with `_` that don't exist at runtime.